test/parallel/test-http-destroyed-socket-write2.js

```diff
@@ -71,7 +77,7 @@ server.listen(0, function() {
 
     assert.strictEqual(req.outputData.length, 0);
     server.close();
-  }));
+  }, 2));
```

Comment:
Hmm was the `'error'` event allowed to be emitted more than once before?

Reply:
No, it was emitted once before, and now it is emitted twice, one because of `write()`, and one because of `'ECONNRESET'`. I could potentially change this line https://github.com/nodejs/node/pull/29227/files#diff-286202fdbdd74ede6f5f5334b6176b5cR302 to be an `else if`, essentially _not_ emitting the error if there is a callback. I think that should fix this issue.

Wdyt?

Reply:
We still have two `'error'` events if callback is not used right? In that case I think this is semver-major.

Reply:
What about setting `this[kErrorEmitted] = true;` for `ECONNRESET` as well?

Reply:
@lpinca updated. Now it should be semver-minor.